### ISSUE TITLE
[Feature] hope to support expand/fold all files

### ISSUE THREAD

**Issue by lizy0329** [EnixCoda/Gitako#92]
hope to support expand/fold all files

**Comment by EnixCoda** [comment]
Hi @lizy0329 

Could you explain the scenario in which you'd like to do that?

BTW, have you noticed that there is `Toggle folders recursively while holding` in the settings?

**Comment by lizy0329** [comment]
Yes, I have noticed that  `Toggle folders recursively while holding` here. But it has to click one by one.
like ![20201107141830.png](https://i.loli.net/2020/11/07/GANdSiKDZeQ9HrV.png) Collapse Folders in vscode

**Comment by EnixCoda** [comment]
I see.

And I believe a detailed scenario in which you'd use that button can convince me into adding the feature better. From that can I tell why the button is really necessary.

**Comment by zzztttkkk** [comment]
After searching, all directories will be expanded and need to be folded manually. It would be much better to have a button.😁

**Comment by EnixCoda** [comment]
@zzztttkkk When folding directories, what are you looking for? Search results under a specific directory or something else? I want to see the reason behind folding them.

**Comment by EnixCoda** [comment]
Folders will get collapsed when the search input gets clear since `v2.7.0`, as somehow an improvement to this.
